boot/bootutil/src/loader.c

```diff
@@ -990,6 +990,19 @@ boot_copy_image(struct boot_loader_state *state, struct boot_status *bs)
         size += this_size;
     }
 
+#if defined(MCUBOOT_SWAP_USING_MOVE)
+    /* When using MCUBOOT_SWAP_USING_MOVE, primary region is larger then the secondary region 
```

Comment:
adjust commit title to: `boot: bootutil: loader: Fix bootstrap copying in swap move mode`